saleor/attribute/tasks.py

```diff
@@ -0,0 +1,23 @@
+from celery.utils.log import get_task_logger
+from django.db.models import Exists, OuterRef, Q
+
+from ..attribute.models import AttributeValue
+from ..celeryconf import app
+from ..product.models import Product, ProductVariant
+from ..product.search import update_products_search_vector
+
+task_logger = get_task_logger(__name__)
+
+
+@app.task
+def update_associated_products_search_vector(attribute_value_pk: int):
+    task_logger.info("Updating associated products search_vector")
+    instance = AttributeValue.objects.get(pk=attribute_value_pk)
```

Comment:
Should we consider a situation when `AttributeValue` is removed before the task was started?

Reply:
Good point 👍